src/tokens/universal.tokens.json

```diff
@@ -1198,37 +970,6 @@
     }
   },
   "effect": {
-    "focus state": {
```

Comment:
Could we keep these - otherwise updating is going to be painful in brave-core :smile: